src/Parameters.jl

```diff
@@ -213,7 +213,7 @@ and the inverse trasnform is the natural logarithm ``f^{-1} ≡ \\log``,
 \\log(Y) = X ∼ 𝒩(μ, σ).
 ```
 """
-transform_to_unconstrained(Π::Normal,    Y) = Y / abs(Π.μ)
+transform_to_unconstrained(Π::Normal,    Y) = (Y - Π.σ) / abs(Π.μ)
```

Comment:
This seems important. Should we put this change into an independent PR so it's easier to see and document?

Reply:
I'm happy to make that PR if you don't have time.

Reply:
Sure!

Reply:
https://github.com/CliMA/ParameterEstimocean.jl/pull/249